x/marker/keeper/keeper_test.go

```diff
@@ -943,3 +946,40 @@ func TestGetAuthority(t *testing.T) {
 	app := simapp.Setup(t)
 	require.Equal(t, "cosmos10d07y265gmmuvt4z0w9aw880jnsr700j6zn9kn", app.MarkerKeeper.GetAuthority())
 }
+
+func TestRemoveIsSendEnabledEntries(t *testing.T) {
+	app := simapp.Setup(t)
+	ctx := app.BaseApp.NewContext(false, tmproto.Header{})
+	markerAddr := markertypes.MustGetMarkerAddress("testcoin").String()
+
+	err := app.MarkerKeeper.AddMarkerAccount(ctx, &markertypes.MarkerAccount{
+		BaseAccount: &authtypes.BaseAccount{
+			Address:       markerAddr,
+			AccountNumber: 23,
+		},
+		AccessControl: []markertypes.AccessGrant{
+			{
+				Address:     "cosmos1w6t0l7z0yerj49ehnqwqaayxqpe3u7e23edgma",
+				Permissions: markertypes.AccessListByNames("deposit,withdraw"),
+			},
+		},
+		Denom:      "testcoin",
+		Supply:     sdk.NewInt(1000),
+		MarkerType: markertypes.MarkerType_Coin,
+		Status:     markertypes.StatusActive,
+	})
+	require.NoError(t, err, "should have added marker")
+	app.BankKeeper.SetSendEnabled(ctx, "testcoin", false)
+	app.BankKeeper.SetSendEnabled(ctx, "nonmarkercoin", false)
+
+	sendEnabledItems := app.BankKeeper.GetAllSendEnabledEntries(ctx)
+	assert.Equal(t, len(sendEnabledItems), 2, "should have 2 items before removal")
+
+	app.MarkerKeeper.RemoveIsSendEnabledEntries(ctx)
+	sendEnabledItems = app.BankKeeper.GetAllSendEnabledEntries(ctx)
+	assert.Equal(t, len(sendEnabledItems), 1, "denom without a marker should only exist")
+	assert.Equal(t, sendEnabledItems[0].Denom, "nonmarkercoin", "denom without a marker should only exist")
+	assert.False(t, app.BankKeeper.IsSendEnabledDenom(ctx, "nonmarkercoin"), "should be in table as false since there is not a marker associated with it")
+	assert.True(t, app.BankKeeper.IsSendEnabledDenom(ctx, "testcoin"), "should not exist in table therefore default to true")
```

Comment:
Good use of `.False` and `.True` here.

The thing being tested is the result of `IsSendEnabledDenom` which returns a boolean. So it makes sense to use those tests here.

Just wanted to point that out. Feel free to resolve this comment whenever.